src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/DefaultCSharpCodeActionProvider.cs

```diff
@@ -29,6 +33,12 @@ internal class DefaultCSharpCodeActionProvider : CSharpCodeActionProvider
             RazorPredefinedCodeFixProviderNames.RemoveUnusedVariable,
         };
 
+        // This is aspirational :)
```

Comment:
Agree this would be a good addition, is there a tracking work item we can link so this doesn't get lost?

Reply:
Feel free to disagree, but to me this is part of https://github.com/dotnet/razor-tooling/issues/6159

In general I figure that issue will either mean we can completely remove the allow list of code actions, because everything is supported, or it means we need to review the allow list after we've added smarts. I would think both of those will pick this up.

We could log a specific issue for "think about which Roslyn code actions should apply in implicit expressions" but it doesn't feel to me like the type of issue that will ever graduate from the backlog, and I don't want to get to the Roslyn "5000+ issues" problem :)

Reply:
Could this comment go into a bit of detail about why we currently only aspire to this?